### ISSUE TITLE
Applying tensorflow lite models

### ISSUE THREAD

**Issue by fracpete** [waikato-ufdl/ufdl-imageclass-app#5]
Allow the user to make use of a model that was built by a previous [job](https://github.com/waikato-ufdl/ufdl-job-launcher-plugins/blob/master/src/ufdl/joblauncher/classify/tensorflow.py):
* user needs to be able to download a model
* user needs to be able to downloaded model for making predictions
* model can be applied to camera feed or image(s) in dataset
* [Android image classification example app](https://github.com/tensorflow/examples/tree/master/lite/examples/image_classification/android)

Depends on https://github.com/waikato-ufdl/ufdl-job-launcher-plugins/issues/5

**Comment by YugPatel31** [comment]
- Added support for TensorFlow Lite & PyTorch Mobile model frameworks.
- Models can be applied to camera feed to produce the top 3 prediction results at each analyzed frame. This option is provided via a button in the camera fragment menu (only available if there is a selected model to use)
- Users are able to change between models & frameworks via bottom sheet selection in the camera fragment.
- Users are provided with an option to let a model auto-classify selected images in the Images Fragment. They will be presented with a dialog where they will be able to choose the model + minimum confidence score required to let the model overwrite the current label.

During the time of development, Jobs weren't functional and hence, the workaround was to rely on models loaded from the assets folder.
